### PR TITLE
feature(REPORT-333396): Bold-Reports 5.3 Release Changes

### PR DESCRIPTION
**Pull request by RamyaManickam1927**
<table>
<tr>
<td>
Description:
</td>
<td>
Bold-Reports 5.3 Release Changes
</td>
</tr>
<tr>
<td>
Analysis:
</td>
<td>
Analyzed about 5.3 release changes in javascript-reporting-controls package
</td>
</tr>
<tr>
<td>
Reason for not identifying earlier:
</td>
<td>
It's a new feature
</td>
</tr>
<tr>
<td>
Is it a breaking issue?
</td>
<td>
No, It's a new feature
</td>
</tr>
<tr>
<td>
Solution description:
</td>
<td>
Include the 5.3 release changes for package javascript-reporting-controls
</td>
</tr>
<tr>
<td>
Areas affected and ensured:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test cases:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test bed sample location:
</td>
<td>
VS, VS Code
</td>
</tr>
</table>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,143 @@
-# Bold Reports JavaScript samples
+# Reports JavaScript samples
```
Comment (karthickthangasamy) [inline README.md:1]: please revert this file changes

File: build/templates/common/index.html [hunk 2]
```diff
@@ -30,12 +22,29 @@
             }
         }
     </script>
+    <!-- Google Tag Manager -->
+    <script>
```
Comment (karthickthangasamy) [inline build/templates/common/index.html:26]: remove the tag manager changes

File: build/templates/report-designer/index.html [hunk 3]
```diff
@@ -22,20 +22,29 @@
...
-    <script type="text/javascript" src="../scripts/common/ej2-data.min.js"></script>
-    <script type="text/javascript" src="../scripts/common/ej2-pdf-export.min.js"></script>
-    <script type="text/javascript" src="../scripts/common/ej2-svg-base.min.js"></script>
-    <script type="text/javascript" src="../scripts/data-visualization/ej2-circulargauge.min.js"></script>
-    <script type="text/javascript" src="../scripts/data-visualization/ej2-lineargauge.min.js"></script>
-    <script type="text/javascript" src="../scripts/data-visualization/ej2-maps.min.js"></script>
-
+    <!-- Google Tag Manager -->
+    <script>
+        (function (w, d, s, l, i) {
```
Comment (karthickthangasamy) [inline build/templates/report-designer/index.html:27]: remove the tag manager scripts

**Review by karthickthangasamy — CHANGES_REQUESTED** [review]

File: build/templates/common/index.html (hunk 2 above)
Comment (RamyaManickam1927) [reply build/templates/common/index.html:25]: Done

File: build/templates/report-designer/index.html (hunk 3 above)
Comment (RamyaManickam1927) [reply build/templates/report-designer/index.html:26]: Done

File: README.md (hunk 1 above)
Comment (RamyaManickam1927) [reply README.md]: Done

File: build/templates/report-designer/index.html [hunk 4]
```diff
@@ -22,20 +22,25 @@
...
-
+    <script>
+        (function (w, d, s, l, i) {
+            w[l] = w[l] || []; w[l].push({
+                'gtm.start':
+                    new Date().getTime(), event: 'gtm.js'
+            }); var f = d.getElementsByTagName(s)[0],
+                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+        })(window, document, 'script', 'dataLayer', 'GTM-K73VBT8');</script>
```
Comment (karthickthangasamy) [inline build/templates/report-designer/index.html:33]: @RamyaManickam1927  please properly remove the tag manager `gtm` scripts in all the places

**Review by karthickthangasamy — CHANGES_REQUESTED** [review]

File: build/templates/report-designer/index.html (hunk 4 above)
Comment (RamyaManickam1927) [reply build/templates/report-designer/index.html]: Done

**Review by karthickthangasamy — APPROVED** [review]

**Pull request merged**